lib/attribute_normalizer/extras.rb

```diff
@@ -1,7 +1,13 @@
+require "attribute_normalizer"
 require "attribute_normalizer/extras/version"
+require "attribute_normalizer/normalizers/gsub_normalizer"
+require "attribute_normalizer/normalizers/squash_normalizer"
+require "attribute_normalizer/normalizers/postal_code_normalizer"
 
-module AttributeNormalizer
-  module Extras
-    # Your code goes here...
-  end
-end
+extras = {
+ gsub:  AttributeNormalizer::Normalizers::GsubNormalizer,
+ postal_code: AttributeNormalizer::Normalizers::PostalCodeNormalizer,
+ squash: AttributeNormalizer::Normalizers::SquashNormalizer
```

Comment:
so i wanted it to be called something different from both - if only there was a verb for removing all whitespace characters from a string